importer/api/__init__.py

```diff
@@ -148,7 +148,7 @@ def import_children(course, element, parent, parent_dpath):
         None
     """
     # pylint: disable=too-many-locals
-    title = element.attrib.get("display_name", "MISSING")
+    title = element.attrib.get("display_name", "...")
```

Comment:
@giocalitri @pdpinch 
Getting issues using unicode string
- unable to create LearningResource in test case ```<exporter.tests.test_export.TestExport testMethod=test_export_resources_to_directory>```
 https://gist.github.com/amir-qayyum-khan/dc97cac3a38326a48450

- haystack issue
https://gist.github.com/amir-qayyum-khan/5fba27592c8020d5f7f8